### PR TITLE
Upgrade to Expo SDK 37

### PR DESCRIPTION
**Pull request by mcallisto**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,13 +12,14 @@
...
-    "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
+    "expo": "^37.0.7",
+    "react": "16.9.0",
+    "react-dom": "16.9.0",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
     "react-proxy": "1.1.8"
   },
   "devDependencies": {
-    "babel-preset-expo": "^5.2.0"
+    "@babel/core": "^7.9.0",
```
Comment (shadaj) [inline package.json]: Why do we need this dependency? Scala.js is still emitting ES5 so not sure why Babel would need to be involved.

**Review by shadaj — COMMENTED** [review]
Just one question, otherwise looks great!

**Comment by mcallisto** [comment]
You are right, is not needed, removed. Can you merge (and squash) like this?

File: package.json (hunk 1 above)
Comment (mcallisto) [reply package.json]: You are right, is not needed, removed. Can you merge (and squash) like this?

**Review by shadaj — APPROVED** [review]

**Pull request merged**